static-build/pages/test/index.tsx

```diff
@@ -89,100 +90,106 @@ const TestPage: FunctionalComponent<Props> = ({ test }: Props) => {
           </section>
         </header>
 
-        {test.subTests && (
-          <main class={$collectionPage}>
-            <section class={$collectionSummary}>
-              <h3>Why</h3>
-              <article
-                dangerouslySetInnerHTML={{ __html: test.html }}
-              ></article>
-              <p class={$well}>
-                Use the (+) card below & tell us if we missed a capability!
-              </p>
-            </section>
+        <main>
+          {test.subTests && (
+            <div class={`${$contentContainer} ${$collectionPage}`}>
```

Comment:
This template has two `<main>`s which kinda made me nervous, but only one ever appears at once. Anyway, I moved the `<main>` to an outer wrapping element.